Client/Frontend/Library/HistoryPanel/HistoryActionables.swift

```diff
@@ -31,7 +31,7 @@ struct HistoryActionablesModel: Hashable {
         self.itemIdentity = itemIdentity
 
         if let imageName = imageName {
-            self.itemImage = UIImage(named: imageName)?.withTintColor(ThemeManager.shared.currentTheme.colours.iconSecondary)
+            self.itemImage = UIImage(named: imageName)?.withTintColor(DefaultThemeManager.shared.currentTheme.colors.iconSecondary)
```

Comment:
In that case we don't pass the theme manager in the init with a defaut param?

Reply:
I wrote a response to this but I have no idea what happened to it.
This class conforms to hashable which complicates things. The easiest path to move forward was to leaving it referencing the singleton. I plan to circle back once we are at the point where we're ready to update the rest of the project to use the new system.